### ISSUE TITLE
Error: /bin/sh: 1: /home/user/.vsdbg/getvsdbg.sh: Permission denied

### ISSUE THREAD

**Issue by metemertkan** [microsoft/vscode-docker#2453]
This is the  permissions of my debug file 
drwxrwxr-x  2 user user 4096 okt 30 14:56 .vsdbg

Does this occur consistently? Yes 

Action: docker-attach
Error type: 126
Error Message: Process '/home/user/.vsdbg/getvsdbg.sh -v latest -r linux-x...' exited with code 126
Error: /bin/sh: 1: /home/user/.vsdbg/getvsdbg.sh: Permission denied

Version: 1.7.0
OS: linux
OS Release: 5.4.0-52-generic
Product: Visual Studio Code
Product Version: 1.50.1
Language: en

<details>
<summary>Call Stack</summary>

```
ChildProcess.<anonymous> extension.bundle.js:46:91411
ChildProcess.emit events.js:223:5
maybeClose child_process.js:1021:16
Socket.<anonymous> child_process.js:430:11
Socket.emit events.js:223:5
Pipe.<anonymous> net.js:664:12
```

</details>

**Comment by bwateratmsft** [comment]
@metemertkan can you check the permissions on that file?

**Comment by bwateratmsft** [comment]
@gregg-miskelly Any ideas what's going on? Is this the error message you might see if vsdbg can't install to the targeted directory?

**Comment by gregg-miskelly** [comment]
I **think** what that error message is saying is that executing getvsdbg.sh failed because permission was denied.

@metemertkan what do you see from `ls -l /home/user/.vsdbg/getvsdbg.sh`?

**Comment by metemertkan** [comment]
Hello @gregg-miskelly thanks for the advice.
It was:
-rw-rw-r-- 1 user user 15112 okt 30 14:56 getvsdbg.sh

I run 'chmod +x' and it worked. I wasn't excepting "Permission Denied" exception.

**Comment by bwateratmsft** [comment]
@gregg-miskelly I'm a knucklehead. We recently refactored our code for fetching vsdbg and I forgot to add the `chmod +x` back in.

**Comment by bwateratmsft** [comment]
This fix has now been released in 1.8.0.
